src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DownloadStationProxy.cs

```diff
@@ -1,121 +1,56 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using NLog;
-using NzbDrone.Common.Extensions;
-using NzbDrone.Common.Http;
-using NzbDrone.Core.Download.Clients.DownloadStation.Responses;
+﻿using System.Collections.Generic;
 
 namespace NzbDrone.Core.Download.Clients.DownloadStation.Proxies
 {
-    public interface IDownloadStationProxy
+    public interface IDownloadStationProxy 
     {
         IEnumerable<DownloadStationTask> GetTasks(DownloadStationSettings settings);
-        Dictionary<string, object> GetConfig(DownloadStationSettings settings);
         void RemoveTask(string downloadId, DownloadStationSettings settings);
         void AddTaskFromUrl(string url, string downloadDirectory, DownloadStationSettings settings);
         void AddTaskFromData(byte[] data, string filename, string downloadDirectory, DownloadStationSettings settings);
+        Dictionary<string, object> GetConfig(DownloadStationSettings settings);
         IEnumerable<int> GetApiVersion(DownloadStationSettings settings);
     }
 
-    public class DownloadStationProxy : DiskStationProxyBase, IDownloadStationProxy
+    public class DownloadStationProxy : IDownloadStationProxy
     {
-        public DownloadStationProxy(IHttpClient httpClient, Logger logger)
-            : base(httpClient, logger)
+        private IDownloadStationTaskProxy _taskProxy;
+        private IDownloadStationInfoProxy _infoProxy;
+
+        public DownloadStationProxy(IDownloadStationTaskProxy taskProxy, IDownloadStationInfoProxy infoProxy)
```

Comment:
A proxy shouldn't call other proxies, a proxy is meant to be a simple interface for mocking so it's easy to mock API calls without having to build the URL each time. Calling another proxy likely means there is too much business logic there. In this case it looks like an unnecessary layer though.